.github/ISSUE_TEMPLATE/project-pitch.yml

```diff
@@ -5,8 +5,35 @@ labels: ['pitch']
 body:
   - type: markdown
     attributes:
-      value: |
-        Thank you for submitting a project to Brainhack Western! Before you
+      value: >
+        Thank you for submitting a project to Brainhack Western! As a project lead,
+        you'll be working with your team of hackers on a neuroscience project in
```

Comment:
"your team of hackers" -> "a team of hackers" to avoid implying they need to come with a team formed